styles.css

```diff
@@ -0,0 +1,216 @@
+  font-size: 20px;
+  padding-left: 10px;
+}
+
+#title {
+  grid-area: title;
+}
+
+#toggle-icon {
+  grid-area: toggle-feed;
+  padding-top: 10px;
+}
+
+#logo {
+  grid-area: logo;
+  padding-left: 24px;
+  padding-top: 20px;
+}
+
+.main-frame {
+  display: grid;
+  grid-template-columns: 0.5fr 260px 2fr 350px 0.5fr;
+  grid-template-rows: 55px 600px;
+  grid-template-areas:
+    ".   left-side    header      right-side   ."
+    ".   left-side    feed        right-side   .";
+}
+
+.left-side {
+  grid-area: left-side;
+  display: grid;
+  grid-template-rows: 70px 400px;
+  grid-template-areas:
+    "logo"
+    "navigation";
+  border-right: solid grey 2px;
+  /* background-color: lightsteelblue; */
+}
+
+.navigation {
+  display: grid;
+  grid-area: navigation;
+  grid-template-columns: 40px 110px;
+  grid-template-areas: "nav-icons nav-list";
+  padding-left: 26px;
+  padding-top: 25px;
+}
+
+.nav-icons {
+  grid-area: nav-icons;
+  display: grid;
+  grid-template-rows: repeat(7, 30px);
+  gap: 23px;
+}
+
+.nav-list {
+  grid-area: nav-list;
+}
+
+.nav-list ul {
+  list-style-type: none;
+}
+
+li > a {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 20px;
+  text-decoration: none;
+  color: blue;
+}
+
+li {
+  margin-bottom: 30px;
+}
+
+ul {
+  padding: 0;
+  margin-top: 6px;
+}
+
+#Tweeter {
+  background-color: rgb(30, 27, 226);
+  width: 200px;
+  height: 40px;
+  border-radius: 50px;
+  text-align: center;
+  color: aliceblue;
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+  margin-left: 22px;
+}
+
+.feed {
+  grid-area: feed;
+  /* background-color: lightgrey; */
+}
+
+.composer {
+  display: grid;
+  grid-template-columns: 70px 180px 1fr 85px;
+  grid-template-rows: 70px 50px;
+  grid-template-areas:
+    "pic-message    pic-message     pic-message    pic-message "
+    "     .       composer-icons       .     tweet-composer";
+  border-bottom: solid grey 2px;
+}
+
+#pic-message {
+  grid-area: pic-message;
+  display: grid;
+  grid-template-columns: 80px 1fr;
+  grid-template-areas: "profile-pic   default-message";
+  border-bottom: dotted grey 2px;
+  margin-left: 5px;
+  margin-right: 9px;
+}
+
+#profile-pic {
+  object-fit: cover;
+  border-radius: 50%;
+  width: 50px;
+  padding: 10px;
+}
+
+#defualt-message p {
+  grid-area: default-message;
+}
+
+.composer-icons {
+  grid-area: composer-icons;
+}
+
+.composer-icons img {
+  height: 21px;
+  padding: 5px;
+}
+
+#Tweeter-composer {
+  grid-area: tweet-composer;
+  background-color: rgb(30, 27, 226);
+  width: 75px;
+  height: 36px;
+  border-radius: 50px;
+  margin-top: 6px;
+  text-align: center;
+  color: aliceblue;
+  font-weight: bold;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.right-side {
+  grid-area: right-side;
+  background-color: mediumturquoise;
+  border-left: solid grey 2px;
+}
+
+@media screen and (max-width: 1000px) {
```

Comment:
This is the first thing I noticed when I opened it up in the browser. So cool and makes for a good user experience! was this possible because you made it a grid? 
If so, very smart move